My-Spec-Tutorial.package/MyFirstWindow.class/class/open.st

```diff
@@ -0,0 +1,4 @@
+opening
+open
```

Comment:
It's nice to abstract the (in this case Spec) implementation details and give users a simple message here.